src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -340,14 +340,15 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       case None => Put
     }
 
-    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } }
-
-    // don't leak any sensitive data
-    RequestComplete(NoContent)
-  }.recover {
+    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } } map { response =>
```

Comment:
@davidangb @dvoet After investigating I don't see how this call to rawls will ever NOT timeout (unless it returns an error quickly like you said). When I tested this yesterday with between 80-90 protected workspaces in dev, it was generating ~23,000 queries to update all of the rawls groups, taking a total of ~38 minutes (granted, local is a lot slower than dev...). The dev DB was in a horrible state for testing whitelist syncing so today I cleaned it all up and made just 1 protected workspace. Syncing the whitelist now doesn't time out, but it generates about 280 queries- which is in line with the 23k @ 80 workspaces I saw yesterday. 

I tested a few other things:

adding a user as a reader on a non-protected workspace: ~135 queries
adding a user as a reader on a protected workspace: ~870 queries

So I think this should spawn another tech debt story which is to refactor the group update queries. As for this PR (and GAWB-843)- I'm happy to clean this up based on the other comments.

I also attached the Slick log output for updating the whitelist with 1 protected workspace.

[whitelist_sync_output.txt](https://github.com/broadinstitute/firecloud-orchestration/files/362086/whitelist_sync_output.txt)